configs/ARM/BeagleBone/Rostock-4DOF-CRAMPS/rostock-3dof.py

```diff
@@ -0,0 +1,96 @@
+# HAL file for BeagleBone + TCT paralell port cape with 5 steppers and 3D printer board
+import os
+
+from machinekit import rtapi as rt
+from machinekit import hal
+from machinekit import config as c
+
+from fdm.config import velocity_extrusion as ve
+from fdm.config import base
+from fdm.config import storage
+from fdm.config import motion
+import cramps as hardware
+import delta as lineardelta
+
+# initialize the RTAPI command client
+rt.init_RTAPI()
+# loads the ini file passed by linuxcnc
+c.load_ini(os.environ['INI_FILE_NAME'])
+
+motion.setup_motion(kinematics='lineardeltakins')
+lineardelta.setup_linearDeltaKins(c)
+
+hardware.init_hardware()
+storage.init_storage('storage.ini')
+
+# reading functions
+hardware.hardware_read()
+hal.addf('motion-command-handler', 'servo-thread')
+hal.addf('motion-controller', 'servo-thread')
+
+numFans = c.find('FDM', 'NUM_FANS')
+numExtruders = c.find('FDM', 'NUM_EXTRUDERS')
+numLights = c.find('FDM', 'NUM_LIGHTS')
+
+# Axis-of-motion Specific Configs (not the GUI)
+ve.velocity_extrusion(extruders=numExtruders, thread='servo-thread')
+# X [0] Axis
+base.setup_stepper(section='AXIS_0', axisIndex=0, stepgenIndex=0, thread='servo-thread')
+# Y [1] Axis
+base.setup_stepper(section='AXIS_1', axisIndex=1, stepgenIndex=1, thread='servo-thread')
+# Z [2] Axis
+base.setup_stepper(section='AXIS_2', axisIndex=2, stepgenIndex=2, thread='servo-thread')
+
+# Extruder, velocity controlled
+for i in range(0, numExtruders):
+    base.setup_stepper(section='EXTRUDER_%i' % i, stepgenIndex=3,
+                       axisIndex=(i+3), thread='servo-thread')
+
+# Extruder Multiplexer
+base.setup_extruder_multiplexer(extruders=numExtruders, thread='servo-thread')
+
+# Stepper Multiplexer
```

Comment:
following lines are commented out. Is the functionality still around when uncommented? If not, can you remove the commented lines?

Reply:
Basically this file should be in sync with
configs/ARM/BeagleBone/MendelMax-CRAMPS/mendelmax.py 
i just added lineardelta configuration

fixed.